components/site/src/link_checking.rs

```diff
@@ -171,7 +171,8 @@ pub fn check_external_links(site: &Site) -> Vec<String> {
     }
 
     println!(
-        "Checking {} external link(s). Skipping {} external link(s).{}",
+        "Checking {} unique link(s) out of {} total external link(s) on all pages. Skipping {} external link(s).{}",
```

Comment:
I don't know if we need the distinction between unique vs total. We can keep the current message but show the unique number